config/OPM-Reconfig.cfg

```diff
@@ -0,0 +1,499 @@
+			avg = 0
+			stddev = 0.05
+		}
+
+ 		ascNode
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		periapsis
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		orbitPhase
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		asteroidTypes
+		{
+			key = 0.75 PotatoRoid
+			key = 0.20 CaAsteroidCarbon
+			key = 0.05 CaAsteroidMetal
+		}
+	}
+
+ 	ASTEROIDGROUP
+	{
+		name = sarnusInner
+		title = #autoLOC_CustomAsteroids_OPM_GroupInnerObj
+
+ 		centralBody = Sarnus
+
+ 		spawnRate = 0.15
+
+ 		orbitSize
+		{
+			dist = Uniform
+			min = Ratio(Sarnus.rad, 1.89)
+			max = Ratio(Ovok.sma, 1)
+		}
+
+ 		eccentricity
+		{
+			dist = Rayleigh
+			avg = 0
+			stddev = 0.05
+		}
+
+ 		inclination
+		{
+			dist = Rayleigh
+			avg = 0
+			stddev = 0.05
+		}
+
+ 		ascNode
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		periapsis
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		orbitPhase
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		asteroidTypes
+		{
+			key = 0.75 PotatoRoid
+			key = 0.20 CaAsteroidCarbon
+			key = 0.05 CaAsteroidMetal
+		}
+	}
+
+ 	ASTEROIDGROUP
+	{
+		name = urlumInner
+		title = #autoLOC_CustomAsteroids_OPM_GroupInnerObj
+
+ 		centralBody = Urlum
+
+ 		spawnRate = 0.15
+
+ 		orbitSize
+		{
+			dist = Uniform
+			min = Ratio(Urlum.rad, 1.89)
+			max = Ratio(Priax.sma, 1)
+		}
+
+ 		eccentricity
+		{
+			dist = Rayleigh
+			avg = 0
+			stddev = 0.05
+		}
+
+ 		inclination
+		{
+			dist = Rayleigh
+			avg = 0
+			stddev = 0.05
+		}
+
+ 		ascNode
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		periapsis
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		orbitPhase
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		asteroidTypes
+		{
+			key = 0.10 CaAsteroidCarbon
+			key = 0.90 CaAsteroidIcy
+		}
+	}
+
+ 	ASTEROIDGROUP
+	{
+		name = neidonInner
+		title = #autoLOC_CustomAsteroids_OPM_GroupInnerObj
+
+ 		centralBody = Neidon
+
+ 		spawnRate = 0.15
+
+ 		orbitSize
+		{
+			dist = Uniform
+			min = Ratio(Neidon.rad, 1.89)
+			max = Ratio(Thatmo.sma, 1)
+		}
+
+ 		eccentricity
+		{
+			dist = Uniform
+			min = 0.1
+			max = 0.4
+		}
+
+ 		inclination
+		{
+			dist = Gaussian
+			avg = Ratio(Thatmo.inc, 1)
+			stddev = 0.5
+		}
+
+ 		ascNode
+		{
+			dist = Gaussian
+			avg = Ratio(Thatmo.lan, 1)
+			stddev = 0.5
+		}
+
+ 		periapsis
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		orbitPhase
+		{
+			dist = Uniform
+			min = 0
+			max = 360
+		}
+
+ 		asteroidTypes
+		{
+			key = 0.10 CaAsteroidCarbon
+			key = 0.90 CaAsteroidIcy
+		}
+	}
+	
+ 	ASTEROIDGROUP
+	{
+		name = joolOuter
+		title = #autoLOC_CustomAsteroids_OPM_GroupOuterObj
+
+ 		centralBody = Jool
+
+ 		spawnRate = 0.10
+
+ 		orbitSize
+		{
+			dist = Uniform
+			min = Ratio(Tylo.sma, 2)
+			max = Ratio(Pol.sma, 1)
+		}
+
+ 		eccentricity
+		{
+			dist = Uniform
+			avg = 0.2
+			stddev = 0.5
+		}
+
+ 		inclination
+		{
+			dist = Uniform
+			min = 0
+			max = 180
+		}
```

Comment:
Did you mean:

    inclination
    {
        dist = Isotropic
    }
A uniform distribution in inclination will, counterintuitively, create slightly more orbits near the planet's equatorial plane (both prograde and retrograde) than in any high-inclination plane.

Reply:
There are certain Uniform distribution nodes still remaining in the patch, but 0-180 inclinations were replaced